Formula/c/cody.rb

```diff
@@ -0,0 +1,21 @@
+class Cody < Formula
+  desc "Sourcegraph Cody CLI"
+  homepage "https://sourcegraph.com"
+  url "https://registry.npmjs.org/@sourcegraph/cody/-/cody-5.5.15.tgz"
+  sha256 "2f25357f26e7710b71e4abead3ad9ee3fc5817fbbd6a78cd6726556982bc935a"
+  license "MIT"
+
+  depends_on "node"
+
+  def install
+    ENV["NPM_CONFIG_PREFIX"] = libexec.to_s
+    system "npm", "install", *std_npm_args
+    rm Dir["#{bin}/*"]
+    bin.install_symlink Dir["#{libexec}/bin/*"]
```

Comment:
Based on your experience, do you think that's better to remove it?

Reply:
I'm just uncertain of the benefits. All Homebrew formulae are build in a temp directory anyway, so there would never be any leftovers. I think it's better to give it a try without this.

Reply:
Good to known! Removing `rm Dir["#{bin}/*"]` (just committed removing it), there was not difference when installing, uninstalling, reinstalling between same/different versions, but without `bin.install_symlink Dir["#{libexec}/bin/*"]`,  the binary is available (need to fix PATH manually), so the symlink we need to keep.

Reply:
@SMillerDev , please let me know if more comments or if I should resolve this thread.

Reply:
That is probably an effect of setting the install location to `libexec`, does it work if you just keep the default install location?

Reply:
@SMillerDev Just keeping default local, doesn't work out of the box, since install path doesn't exist natively in the PATH env variable (in truth, it works, but the binary is hidden in the install path, so for some of the users will not "work"), so the symlink still required, but I was able to simplify a little bit, can you please take a look when possible?

Reply:
@SMillerDev , Is this approach viable for moving forward? If not, no worries - I can continue using my personal tap, close this PR, and revisit in the future if Cody CLI aligns better with the project requirements.
The main goal here was to simplify things for other users.

Reply:
I'm resolving this thread, because there is no answers for a long time, and if not able to move forward, I will close the PR.